### PR TITLE
Updated workflow to check vuln risk

### PR DESCRIPTION
**Pull request by sengleson-r7**
added logic to reduce noise for trivial vulnerability remediations

Tested in integrations lab with dev Teams account

### CODE REVIEW

File: workflows/Send_Remediated_Vulnerabilities_Message_in_Microsoft_Teams_from_InsightVM/Send_Remediated_Vulnerabilities_Message_in_Microsoft_Teams_from_InsightVM.icon [hunk 1]
```diff
@@ -668,148 +711,46 @@
...
+                        "value": 4
                       },
                       "type": "binary"
                     },
-                    "expressionText": "1 == 1"
+                    "expressionText": "{{[fa22ff16-8091-42a5-8234-377ac4809ec4].[output]}} > 4"
                   },
-                  "name": "Total Risk Count",
-                  "outputType": "count"
+                  "name": "Risk Threshold",
```
Comment (tharward-r7) [inline workflows/Send_Remediated_Vulnerabilities_Message_in_Microsoft_Teams_from_InsightVM/Send_Remediated_Vulnerabilities_Message_in_Microsoft_Teams_from_InsightVM.icon:752]: Might make more sense as i.e. "Any over risk threshold?" for a boolean.

**Review by tharward-r7 — DISMISSED** [review]
Looks good, I like the option; one small comment on the vuln boolean loop output name but not blocking.

**Comment by cmchugh-r7** [comment]
should this technically be a major version bump? almost seems like a "breaking" change since it's no longer posting for certain vulns. Also +1 on tristan's comment

**Comment by sengleson-r7** [comment]
> should this technically be a major version bump? almost seems like a "breaking" change since it's no longer posting for certain vulns. Also +1 on tristan's comment
Good call!
* Major version bumped
* Re-named loop output variable & re-tested to confirm it still works

**Review by cmchugh-r7 — APPROVED** [review]

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**